hysds_commons/request_utils.py

```diff
@@ -65,35 +65,40 @@ def post_requests_json_response(url, **kwargs):
     return requests_json_response("POST", url, **kwargs)
 
 
-def post_scrolled_json_responses(url, es_url, generator=False, **kwargs):
+def post_scrolled_json_responses(url, es_url, paginate=False, generator=False, **kwargs):
```

Comment:
```suggestion
def post_scrolled_json_responses(url, es_url, generator=False, paginate=False, **kwargs):
```
@NamrataM: thinking that we should move `paginate` after the `generator` kwarg since `generator` existed before the addition of `paginate` and end users may be calling the method without specifying `generator=<var>`.